app/gui/src/presenter/graph.rs

```diff
@@ -720,18 +717,35 @@ impl Graph {
             }));
 
 
+            // === Execution Environment ===
+
+            // FIXME[Procrat]: Delete me once we synchronise the execution environment with the
+            // language server.
+            execution_environment <- view.toggle_execution_environment.map(
+                f_!([model] model.toggle_execution_environment()));
```

Comment:
nit: 
```suggestion
                f_!(model.toggle_execution_environment()));
```
would also work.